README.md

```diff
@@ -137,6 +137,7 @@ func init() {
 ```
 
 **Start a custom segment/subsegment**
+Note that the `xray.BeginSegment` function currently uses only the service name parameter to determine the applicability of any sampling rules.
```

Comment:
Hi can we rephrase this to something like customers using xray.BeginSegment API directly will only be able to evaluate sampling rules based on service name ? It's important to show that this behavior is not applicable for instrumented requests.

Reply:
That is much clearer :)

I'll update it accordingly